climbing_stairs/climbing_stairs.py

```diff
@@ -3,8 +3,12 @@
 import sys
 
 def climbing_stairs(n, cache=None):
-  pass 
-
+  if n < 0:
+    return 0
+  if n == 0:
+    return 1
+  else:
+    return climbing_stairs(n-1) + climbing_stairs(n-2) + climbing_stairs(n-3)
 
```

Comment:
Hey Stefano , Now that the instructor has gone over cache , you could implement that into your code since that would help make it run faster!